bvt/op-opal-fvt.xml

```diff
@@ -323,5 +323,21 @@
             </testcase>
         </test>
 
+        <test>
+            <name>Test Kernel Crash plus full IPL</name>
+            <testcase>
+                <cmd>../ci/source/op_opal_fvt.py OpalKernelCrashTests.test_kernel_crash_kdump_disable</cmd>
+                <exitonerror>no</exitonerror>
+            </testcase>
+        </test>
+
+        <test>
+            <name>Test Crash kernel plus full IPL</name>
```

Comment:
No, both are different.
1. Kernel Crash plus full IPL
This means we are generating kernel crash(OOPS) and immediately we are IPLing the system and not going to crash kernel for collecting crash dump.
2. Crash Kernel plus full IPL
this means kernel crash(OOPS) + Crash kernel boot(For collecting crash dump) + and then full IPL.